query-parser/src/parser_util.rs

```diff
@@ -40,6 +40,7 @@ macro_rules! matches_plain_symbol {
 /// The provided `$body` will be evaluated with `$input` bound to the input stream.
 ///
 /// `$body`, when run, should return a `ParseResult` of the appropriate result type.
+#[macro_export]
```

Comment:
My understanding is: if it can be used in other places as well, then yes. However, indeed in this particular case `def_value_satisfy_parser_fn` could be used instead of `def_parser_fn`, so maybe don't move unless there's a really good reason to?